src/main/java/ca/gc/aafc/dinauser/api/repository/UserRepository.java

```diff
@@ -26,29 +43,12 @@ public UserRepository() {
   }
 
   @Override
-  public DinaUserDto findOne(String id, QuerySpec querySpec) {
-    return service.getUser(id);
+  public DinaUserDto findOne(Serializable id, QuerySpec querySpec) {
+    return service.getUser(id.toString());
   }
 
   @Override
-  public ResourceList<DinaUserDto> findAll(Collection<String> ids, QuerySpec querySpec) {
+  public ResourceList<DinaUserDto> findAll(Collection<Serializable> ids, QuerySpec querySpec) {
     return querySpec.apply(service.getUsers());
   }
-
```

Comment:
Shouldn't the create, save and delete methods still be in the repo? Or are they exposed as http endpoints from the service directly?

Reply:
I realized DinaRepository has them included already.